### PR TITLE
MAT-9444: word wrap long eCQM Title

### PR DESCRIPTION
**Pull request by etan-sb**
## MADiE PR

Jira Ticket: [MAT-9444](https://jira.cms.gov/browse/MAT-9444)
(Optional) Related Tickets:

image

image

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/templates/humanreadable/human_readable_css.ftl [hunk 1]
```diff
@@ -47,6 +47,12 @@
         margin-top: 10px;
     }
 
+    .ecqm_title {
```
Comment (adongare) [inline src/main/resources/templates/humanreadable/human_readable_css.ftl]: don't think we are following this consistently but usually classes with multiple words are hyphenated.

Comment (etan-sb) [reply src/main/resources/templates/humanreadable/human_readable_css.ftl]: I can change to hyphen to make consistent with the file

**Review by adongare — APPROVED** [review]
